index.html

```diff
@@ -40,6 +41,170 @@ <h2>I'am a Software Developer</h2>
+                    <ul class="project-tech mobile-tech">
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">bootstrap</a></li>
+                        <li><a href="#">Ruby</a></li>
+                    </ul>
+                    <button class="project-button mobile-button">See Project</button>
+                </div>
+            </article>
+            <article class="project-card mobile-card">
+                <div class="transparent">
+                    <div class="project-desc mobile-desc">
+                        <h3 class="desc-title">Multi-Post Stories</h3>
+                        <p class="description mobile-description">
+                            A daily selection of privately personalized reads;
+                            no accounts or sign-ups required. has been 
+                            the industry's standard dummy text ever since the 1500s, 
+                            when an unknown printer took a standard dummy text.
+                        </p>
+                    </div>
+                    <ul class="project-tech mobile-tech">
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">bootstrap</a></li>
+                        <li><a href="#">Ruby</a></li>
+                    </ul>
+                    <button class="project-button mobile-button">See Project</button>
+                </div>
+            </article>
+        </section>
+        <section class="about-section">
+            <header class="about-header">
+                <h3>About Me</h3>
+            </header>
+            <article class="about-desc">
+                <p class="description mobile-description">
+                    A daily selection of privately personalized reads;
+                    no accounts or sign-ups required. has been 
+                    the industry's standard dummy text ever since the 1500s, 
+                    when an unknown printer took a standard dummy text.
+                </p>
+                <div class="abt-bottom">
+                    <div class="project-button about-btn">My Resume</div>
+                    <div class="abt-pic">
+                        <img alt="about image illustration" src="images/about.png">
+                    </div>
+                </div>
+                <hr>
+            </article>
+            <ul class="categories">
+                <li class="desc-title lang-title">Languages
+                <ul class="lang-list">
+                    <li>Javascript</li>
+                    <li>Html</li>
+                    <li>Ruby</li>
+                    <li>Css</li>
+                </ul></li>
+                <li class="desc-title lang-title">Frameworks
+                <ul class="lang-list">
+                    <li>Bootstrap</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSPec</li>
+                    <li>CapyBara</li>
+                    <li>Selenium</li>
```

Comment:
- [Optional] Please stick to the font size in the Figma design for the list of languages which is currently `22px` but on your design it is `24px`